.github/workflows/os-zoo.yml

```diff
@@ -29,11 +29,11 @@ jobs:
     steps:
     - name: install packages
       run: |
-        apk --no-cache add build-base perl linux-headers ${{ matrix.cc }}
+        apk --no-cache add build-base perl linux-headers git ${{ matrix.cc }}
 
     - uses: actions/checkout@v3
-    - name: checkout fuzz/corpora submodule
-      run: git submodule update --init --depth 1 fuzz/corpora
+      with:
+        submodules: true
```

Comment:
Do we really want all submodules?

Reply:
This avoids error I've seen in previous runs when doing the submodule checkout explicitly - this is container so not so trivial to fix. Or I can just drop it and the fuzz/corpora won't be tested. IMO does not matter much.

Reply:
I'm leaning towards dropping it.  The fuzzing runs elsewhere, it's not going to provide anything extra here IMO.

Reply:
OK, np.